src/examples/table/TableCaption.tsx

```diff
@@ -19,8 +19,8 @@ import {
 } from '@zendeskgarden/react-tables';
 
 const Example = () => (
-  <div style={{ minWidth: 500 }}>
-    <Table>
+  <div style={{ overflow: 'auto' }}>
```

Comment:
[nit] these could probably all be `overflowX` as that is the only dimension we care about controlling 🤷

Reply:
Nice, will amend right meow.